.github/workflows/ci.yml

```diff
@@ -0,0 +1,167 @@
+on:
+  pull_request: {}
+  push:
+    branches:
+      - main
+    paths-ignore:
+      - "docs/**"
+
+name: Continuous Integration
+
+jobs:
+  lint-workflows:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: cschleiden/actions-linter@v1
+        with:
+          workflows: '[".github/workflows/*.yaml", ".github/workflows/*.yml"]'
+
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: 14
+          cache: 'yarn'
+
+      - name: Install dependencies
+        run: 'yarn run setup'
+
+      - name: Lint Client Source
+        run: |
+          cd packages/client
+          yarn lint --no-fix
+      
+      - name: Lint Server Source
+        run: |
+          cd packages/server
+          yarn lint
+
+  test-server:
```

Comment:
This job is currently failing, which is expected. See the PR description for details about what needs to be done here.